Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/event/dao/DatabaseEventLoggerDao.java

```diff
@@ -149,4 +147,9 @@ private void transactionRollback(final Transaction tx) {
         }
     }
 
+    protected SessionFactory getSessionFactory() {
```

Comment:
I understand this method calls hibernatUtil.getSessionFactory. Should we consolidate it or refactoring?